bundles/org.openhab.binding.deconz/src/main/java/org/openhab/binding/deconz/internal/action/GroupActions.java

```diff
@@ -59,8 +60,9 @@ public class GroupActions implements ThingActions {
     private @Nullable GroupThingHandler handler;
 
     @RuleAction(label = "@text/action.create-scene.label", description = "@text/action.create-scene.description")
-    public @ActionOutput(name = NEW_SCENE_ID_OUTPUT, type = "java.lang.Integer") Map<String, Object> createScene(
-            @ActionInput(name = "name", label = "@text/action.create-scene.name.label", description = "@text/action.create-scene.name.description") @Nullable String name) {
+    public @ActionOutputs({
```

Comment:
I thought the discussion lead to the point that `@ActionOutputs` was not used and had no purpose?

Reply:
Nevermind:
https://github.com/openhab/openhab-addons/issues/17504#issuecomment-2439931554